src/main/java/com/google/sps/servlets/Report.java

```diff
@@ -19,7 +19,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Entity newReport = new Entity("Report");
     newReport.setProperty("type", request.getParameter("report-type"));
     newReport.setProperty("report", request.getParameter("report"));
-    newReport.setProperty("user-email", request.getParameter("email"));
+    newReport.setProperty("user", request.getParameter("email"));
```

Comment:
why did you change the name? I feel like user-email or email is more intuitive

Reply:
the dash became and issue when trying to pull the attribute because I am sending the entities over rather than making a whole new object. I can just change it to email